e2e/LongHaul/prerequisites/LongHaulSetup.ps1

```diff
@@ -255,7 +263,7 @@ $longhaulModuleConnectionString = az iot hub module-identity connection-string s
 $keyvaultKvps = @{
     # Environment variables for IoT Hub E2E tests
     "IOTHUB-LONG-HAUL-DEVICE-CONNECTION-STRING" = $longhaulDeviceConnectionString;
-    "IOTHUB-LONG-HAUL-MODULE-CONNECTION-STRING" = $longhaulModuleConnectionString;
```

Comment:
I'd recommend having both variations of the module identity so that we can test both flows